openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1191,29 +1193,115 @@ Cache-Control: no-store
 }
 ```
 
+# Notification Endpoint {#notification_endpoint}
+
+This endpoint is used by the Wallet to notify the Credential Issuer of certain events for issued Credentials. These events enable the Credential Issuer to take subsequent actions after issuance. The Credential Issuer needs to return the one or more `notification_id` parameters in the Credential Response or the Batch Credential Response for the Wallet to be able to use this Endpoint. Support for this endpoint is OPTIONAL.
```

Comment:
@pmhsfelix to suggest a text to make it more explicit that the support for this endpoint is optional for the wallet.

Reply:
Suggested text, to insert after "...to be able to use this Endpoint"
```
The Wallet SHOULD notify the issuer of the relevant events by using this endpoint, since this can improve overall functionality and user experience, however this notification is not mandatory.
Namely, the Issuer cannot assume that a notification will ever be sent for any issued credential.
```

Reply:
How about below? cc @jogu

```suggestion
This endpoint is used by the Wallet to notify the Credential Issuer of certain events for issued Credentials. These events enable the Credential Issuer to take subsequent actions after issuance. The Credential Issuer needs to return the one or more `notification_id` parameters in the Credential Response or the Batch Credential Response for the Wallet to be able to use this Endpoint. Support for this endpoint is OPTIONAL. The Issuer cannot assume that a notification will be sent for any issued credential since this Endpoint is not mandatory for the Wallet.
```

Reply:
@Sakurann WDYT about this rewording

> The Issuer should not assume that a notification will be dispatched for every issued Credential, as the use of this endpoint is not mandatory for the Wallet.

Reply:
I'm okay with with any of these suggestions, although if we go with @peppelinux 's then I would replace 'should not'  with 'MUST NOT'.